net.sourceforge.efaxgtk.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: net.sourceforge.efaxgtk
+runtime: org.gnome.Platform
```

Comment:
do you really need the gnome platform?

Freedesktop SDK contains Gtk3.

Reply:
I'll give it a shot.

Reply:
That worked, thanks!